### PR TITLE
Zijie lint fix for reports component Phase 4

### PR DESCRIPTION
**Pull request by iTvX**
# Description
Lint fix for Reports component, this is phase 4.

## Files changed

- src/components/Reports/TotalReport/TotalPeopleReport.jsx
- src/components/Reports/TotalReport/TotalProjectReport.jsx
- src/components/Reports/TotalReport/TotalReportBarGraph.jsx
- src/components/Reports/TotalReport/TotalTeamReport.jsx

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Go to Reports,  Check functionality is the same as before.
- Check dashboard - Report - Total Report is normal.(Please record video)
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record video)

Optional:
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/TotalReport/TotalPeopleReport.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalProjectReport.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalReportBarGraph.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalTeamReport.jsx`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

Screenshot:
CleanShot 2023-09-30 at 12 03 49@2x

### CODE REVIEW

**Review by Haoj1 — DISMISSED** [review]
@iTvX There is no lint error when running the tests on my end. Good job!

**Comment by astrjc0326** [comment]
I logged in as Admin and checked the functionalities inside the Report. All functionalities were working without any bugs. The ESLint check passed without any errors.

Screen Shot 2023-09-30 at 10 44 31 PM

**Review by Bhagyashree444 — APPROVED** [review]
Code passes ESLint checks. Also the Report functionality remains consistent. Great Job!

Screenshot (177)

**Review by luisarevalo21 — APPROVED** [review]
Hello @iTvX ,

I tested your PR and works as intended. Great work! 
Running all the eslint shows no errors and the applications runs normally. Great work
Screenshot 2023-10-02 at 7 46 08 PM

**Review by vikrambadhan — APPROVED** [review]
Hi, I tested your PR and it works as intended!

Week6 - PR1

**Comment by DavidC0126** [comment]
Hi @iTvX I tested the PR and it works as intended!

Screenshot 2023-10-03 at 11 08 01 PM

**Review by Shubhu0 — DISMISSED** [review]
Hi @iTvX Tested the Pr and it is working as intended. No errors in Lint as well

image

**Review by savio-henrique — APPROVED** [review]
Hi @iTvX , I've tested as an Admin and everything worked as intended. As you've said, the code pass without errors in ESLint! 
terminal

**Review by yi-lin-1234 — DISMISSED** [review]
Hello, functionality of the TotalReport pages is the same as before. And, the eslint fix worked as expected. Great work!
Screenshot 2023-10-22 at 4 23 15 PM

**Review by Chuehleo — APPROVED** [review]
Functionalities in the Reports tab works perfectly without any bugs. The ESLint test passed without any errors. Great job!
Screenshot 2023-10-26 000909

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import { Link } from 'react-router-dom';
-import moment from 'moment';
+// eslint-disable-next-line no-unused-vars
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:2]: Same comment as [here](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1356/files#r1384348686)

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 2]
```diff
@@ -75,35 +78,27 @@ const TotalPeopleReport = props => {
...
-    const groupedEntries = Object.entries(groupByTimeRange(allTimeEntries, timeRange));
-    let summaryOfTime = [];
-    groupedEntries.forEach(element => {
-      const groupedUsersOfTime = Object.values(sumByUser(element[1], 'userId'));
-      const contributedUsersOfTime = filterTenHourUser(groupedUsersOfTime);
-      summaryOfTime.push({ timeRange: element[0], usersOfTime: contributedUsersOfTime });
-    });
-    return summaryOfTime;
-  };
+  // const { userProfiles } = props;
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:93]: We don't allow codes in comments, just remove it.

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 3]
```diff
@@ -116,27 +111,70 @@ const TotalPeopleReport = props => {
...
+    const summaryOfTime = [];
+    groupedEntries.forEach(element => {
+      const groupedUsersOfTime = Object.values(sumByUser(element[1], 'userId'));
+      const contributedUsersOfTime = filterTenHourUser(groupedUsersOfTime);
+      summaryOfTime.push({ timeRange: element[0], usersOfTime: contributedUsersOfTime });
+    });
+    return summaryOfTime;
+  };
+
+  // const { startDate, endDate } = props;
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:125]: Same as above.

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 4]
```diff
@@ -116,27 +111,70 @@ const TotalPeopleReport = props => {
...
+    const sumData = groupedDate.map(range => {
+      return {
+        label: range.timeRange,
+        value: range.usersOfTime.length,
+      };
+    });
+    return sumData;
+  };
+
+  // const { endDate, startDate } = props;
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:153]: Same as above.

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 5]
```diff
@@ -116,27 +111,70 @@ const TotalPeopleReport = props => {
...
         setShowMonthly(true);
       }
-      if (props.startDate.getFullYear() !== props.endDate.getFullYear()) {
+      if (startDate.getFullYear() !== endDate.getFullYear()) {
         setShowYearly(true);
       }
     }
   };
 
+  // const { startDate, endDate } = props;
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:170]: Same as above.

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx [hunk 6]
```diff
@@ -246,6 +257,7 @@ const TotalPeopleReport = props => {
         </div>
         {allPeople.length ? (
           <div className="total-detail">
+            {/* eslint-disable-next-line no-unused-vars */}
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalPeopleReport.jsx:260]: Same as above.

File: src/components/Reports/TotalReport/TotalProjectReport.jsx [hunk 7]
```diff
@@ -1,15 +1,15 @@
 import { Link } from 'react-router-dom';
-import moment from 'moment';
+// eslint-disable-next-line no-unused-vars
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalProjectReport.jsx:2]: Same as above.

File: src/components/Reports/TotalReport/TotalReportBarGraph.jsx [hunk 8]
```diff
@@ -1,18 +1,21 @@
+// eslint-disable-next-line no-unused-vars
```
Comment (EvianTan) [inline src/components/Reports/TotalReport/TotalReportBarGraph.jsx:1]: Same as above.

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx (hunk 2 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalPeopleReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx (hunk 3 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalPeopleReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx (hunk 4 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalPeopleReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx (hunk 5 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalPeopleReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalPeopleReport.jsx (hunk 6 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalPeopleReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalProjectReport.jsx (hunk 7 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalProjectReport.jsx]: fixed

File: src/components/Reports/TotalReport/TotalReportBarGraph.jsx (hunk 8 above)
Comment (iTvX) [reply src/components/Reports/TotalReport/TotalReportBarGraph.jsx]: fixed

**Review by EvianTan — APPROVED** [review]

**Pull request merged**